integration_tests/05_docker_placebo_stall.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+my_dir="$(dirname "$0")"
+source "$my_dir/header.sh"
+
+testground plan import --from plans/placebo
+testground build single --builder docker:go --plan placebo | tee build.out
+export ARTIFACT=$(awk -F\" '/generated build artifact/ {print $8}' build.out)
+docker tag $ARTIFACT testplan:placebo
+
+testground healthcheck --runner local:docker --fix
+testground run single --runner local:docker --builder docker:go --use-build testplan:placebo --instances 2 --plan placebo --testcase stall &
+sleep 20
```

Comment:
This just seems like it might be flaky. Maybe use the `timeout` command?

Reply:
Right, I will update with a while loop that waits for containers to appear!